src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private static Random randomColor = new Random();
```

Comment:
Avoid using static variables unless they are constants. Here, the `randomColor` should not be static as this method could be called on an instance of `ColorSupplier`. Consider making `randomColor` a non-static field.